src/index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+      <div class="card__title" >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="card__product-code" >Product code: 195434</div>
+      <div class="card__rating rating">
+        <div class="stars rating__stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rating__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price price">
+        <div class="price__name">Price:</div>
+        <div class="price__number">$2,199</div>
+      </div>
+      <button class="card__button button" data-qa="hover">
+        <a class="button__link" href="#">buy</a>
```

Comment:
```suggestion
        buy
```

Reply:
but in "Requirements" this task "add data-qa="hover" attribute to the link BUY", so I supposed that it means <a>. Doesn't it?